.github/workflows/check-clang-format.yml

```diff
@@ -50,6 +58,80 @@ jobs:
       - name: Validate ClangFormat configuration files
         run: task --silent clang-format:validate
 
+  check-output:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install Task
+        uses: arduino/setup-task@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          version: 3.x
+
+      - name: Set environment variables
+        run: |
+          # See: https://docs.github.com/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable
+          if [[ "${{ github.event.inputs.clang-format-version }}" == "" ]]; then
+            echo "CLANG_FORMAT_VERSION=$(task clang-format:get-version)" >> "$GITHUB_ENV"
+          else
+            echo "CLANG_FORMAT_VERSION=${{ github.event.inputs.clang-format-version }}" >> "$GITHUB_ENV"
+          fi
+          echo "CLANG_FORMAT_INSTALL_PATH=${{ runner.temp }}/clang-format" >> "$GITHUB_ENV"
+          echo "WORKING_FOLDER=${{ runner.temp }}" >> "$GITHUB_ENV"
+
+      - name: Download ClangFormat
+        id: download
+        uses: MrOctopus/download-asset-action@1.0
+        with:
+          repository: arduino/clang-static-binaries
```

Comment:
This is amazing

Reply:
I love that `arduino/clang-static-binaries` repo!